VulnerableApplication/Controllers/HomeController.cs

```diff
@@ -30,5 +30,8 @@ public ActionResult Contact()
         public string Hello() {
         return "this is not on the coding rules.";
         }
+        
+        public static string ChangePassword = "PassWord917231";
```

Comment:
[VULNERABILITY] Make this field 'private' and encapsulate it in a 'public' property.